docs/learn/learn-guides-bounties.md

```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
 [Bounties page](https://polkadot.js.org/apps/#/bounties). Only after the curators claim their
 candidacy the bounty will show as "active" on the main Bounty page.
 
+:::info Curator assignment must execute after bounty funding
+
+Curator assignment must take place after the bounty has been funded, and approved bounties can only
```

Comment:
"can only be funded"  or "are only funded"?

Also, at the beginning of next paragraph, you mention that they are funded (through the treasury) at the end of current spending period, where as here you mention they "can be funded after a spend period has been completed".

We may have to rephrase that

Reply:
changed